### ISSUE TITLE
About output "Julia types" from Python functions.

### ISSUE THREAD

**Issue by martinmestre** [JuliaPy/PythonCall.jl#574]
Hi!
I would like to know if it is correct that all the outputs of python functions have julia type` Py`. 
Is it possible to specify more the type in such a way to use multiple dispatch depending on the python type? 
If not, which is the correct way to assign a specific julia type instead of `Py`?
For example, I am referring to this [python class](http://gala.adrian.pw/en/latest/api/gala.coordinates.GreatCircleICRSFrame.html).
Thank you.

**Comment by cjdoris** [comment]
Yes, PythonCall outputs `Py` from most functions that return some sort of Python object. Hence multiple dispatch on the Python type is not directly possible. For specific use-cases you can hack it yourself with something like:
```julia
foo(x::Py) = foo(x, Val(Symbol(x.__class__.__module__, ".", x.__class__.__name__)))
foo(x::Py, ::Val{Symbol("foo.bar")}) = ...
foo(x::Py, ::Val{Symbol("foo.baz")}) = ...
```
